### PR TITLE
246 accuracy fix

### PR DESCRIPTION
**Pull request by Elforama**
Resolves: https://github.com/Greenstand/treetracker-android/issues/246

### CODE REVIEW

File: app/src/main/java/org/greenstand/android/TreeTracker/fragments/MapsFragment.kt [hunk 1]
```diff
@@ -167,15 +167,10 @@ class MapsFragment : androidx.fragment.app.Fragment(), OnClickListener, OnMarker
...
 
-        var v: View? = view
-
-        try {
-            v = inflater.inflate(R.layout.fragment_map, container, false)
-        } catch (e: InflateException) {
-            Timber.d(e.localizedMessage);
-        }
-
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
```
Comment (mckornfield) [inline app/src/main/java/org/greenstand/android/TreeTracker/fragments/MapsFragment.kt]: So the fix was to use a different lifecycle method for the rest of this?

File: app/src/main/java/org/greenstand/android/TreeTracker/fragments/MapsFragment.kt [hunk 2]
```diff
@@ -195,10 +190,9 @@ class MapsFragment : androidx.fragment.app.Fragment(), OnClickListener, OnMarker
 
         (activity as AppCompatActivity).supportActionBar!!.setDisplayHomeAsUpEnabled(false)
 
-        val fab = v!!.addTreeButton
-        fab.setOnClickListener(this)
-        if (BuildConfig.BUILD_TYPE === "dev") {
-            fab.setOnLongClickListener(this)
+        addTreeButton.setOnClickListener(this)
+        if (FeatureFlags.DEBUG_ENABLED) {
```
Comment (mckornfield) [inline app/src/main/java/org/greenstand/android/TreeTracker/fragments/MapsFragment.kt]: Good cleanup

File: app/src/main/java/org/greenstand/android/TreeTracker/fragments/MapsFragment.kt (hunk 1 above)
Comment (Elforama) [reply app/src/main/java/org/greenstand/android/TreeTracker/fragments/MapsFragment.kt]: Seems like that's what was needed.

**Comment by deepwinter** [comment]
Looks fine to me!

**Pull request merged**